packages/ui/src/components/Typography/title.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import type { TitleProps as DefaultTitleProps } from 'antd/lib/typography/Title'
+import Base from 'antd/lib/typography/Base'
+import classNames from 'classnames'
+
+declare const TITLE_LEVELS: [1, 2, 3, 4, 5, 6]
+declare const TITLE_TYPES: ['inverted']
+export type TypographyTitleProps = Pick<DefaultTitleProps, 'ellipsis' | 'onClick' | 'title' | 'children'> & {
+    type?: typeof TITLE_TYPES[number]
+    level?: typeof TITLE_LEVELS[number]
+}
+
+const TYPOGRAPHY_CLASS_PREFIX = 'condo-typography'
```

Comment:
The same constant is in the file `packages/ui/src/components/Typography/constants.ts`